resources/testProfileData.example.ts

```diff
@@ -2,14 +2,14 @@ import { IProfile } from "@brightside/imperative";
 
 export const profile: IProfile = {
     type : "zosmf",
-    host: "",
-    port: 0,
-    user: "",
-    password: "",
```

Comment:
I'd suggest not changing the example file, but rather creating a `testProfileData.ts` with the desired information.

We should document this better (maybe within this same file?)

Reply:
Additionally you should try and remove this file from this PR sooner rather later as it contains sensitive data. Maybe close this and submit another